### ISSUE TITLE
Proposed redrafting of snippet advice for rejecting requests as vexatious

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/whatdotheyknow-theme#850]
Current text:

>I would like to ask you to reassess my request in light of the fact that the information I have requested would have a serious purpose and value (as per the ICO guidance at https://ico.org.uk/media/for-organisations/documents/1198/dealing-with-vexatious-requests.pdf).
>
>I assure you that my request was not intended to be vexatious.

Proposed:

>I would like to ask you to reassess my request in light of the fact that it has a serious purpose and value. 
>
>I note the Information Commissioner advises that the burden of responding to the request should be balanced against the request's purpose and value (as per the ICO guidance at https://ico.org.uk/media/for-organisations/documents/1198/dealing-with-vexatious-requests.pdf).
>
>I assure you that my request was not intended to be vexatious.

https://www.whatdotheyknow.com/admin/snippets/20/edit

This suggestion arises from [a review of the use of this snippet to-date](https://www.whatdotheyknow.com/search/%22reassess%20my%20request%20in%20light%20of%20the%20fact%20that%20the%20information%20I%20have%20requested%20would%20have%20a%20serious%20purpose%20and%20value%22/all).  Those are cases where the issue isn't that the request is being deemed vexatious due to not having a serious purpose and value at all, but that the purpose and value is claimed not to outweigh the burden on the authority. 

I am slightly wary about making this edit as I'm not entirely sure of all the circumstances in which this snippet would be offered. The summary of the circumstances given is:

>The authority has refused your request on the grounds that it is vexatious, but your request is not trivial, or is of demonstrable significance to you. 

I think the proposed advice should appear if there is a "yes" answer to:

> Has the authority stated that providing the information you have asked for would cause them a disproportionate or unjustified level of disruption, irritation or distress, or 'place an unreasonable burden' on them?

**Comment by RichardTaylor** [comment]
Advice on this subject which appears on /help/unhappy includes:

>A [Court of Appeal case](http://www.bailii.org/ew/cases/EWCA/Civ/2015/454.html) stated that a request may not be vexatious if "the information sought would be of value to the requester or to the public or any section of the public". Consider requesting an internal review, citing [the ICO guidance](https://ico.org.uk/media/for-organisations/documents/1198/dealing-with-vexatious-requests.pdf) and stating the value this information would have if released. 

A reference to this could be included in the snippet too.

**Comment by RichardTaylor** [comment]
I wanted to check to what extent the position really is that a FOI request can be refused as vexatious merely due to the burden on the authority. The [Dransfield Court of Appeal case from 2015](http://www.bailii.org/ew/cases/EWCA/Civ/2015/454.html) suggests a very high threshold for vexatiousness and doesn't appear to support such rejections.

Paras 69-73 of the [ICO guidance](https://ico.org.uk/media/for-organisations/documents/1198/dealing-with-vexatious-requests.pdf) deal with this situation and propose a threshold of "a grossly oppressive burden", this appears to be derived from the case of Independent Police Complaints Commissioner vs The Information Commissioner(EA/2011/0222) which is summarised in the ICO guidance, which includes a quote apparently from the judgment :

>A request may be so grossly oppressive in terms of the resources and time demanded by compliance as to be vexatious, regardless of the intentions or bona fides of the requester.

that's only at the tribunal level though, what higher courts might make of that is not yet fully known, but the Dransfield case suggests it might not fully support it. 

I have personally had a request I considered to be of significant public interest rejected as vexatious and have written about it at:

http://www.rtaylor.co.uk/cambs-police-feb-papers-foi-vexatious.html

**Comment by RichardTaylor** [comment]
I think this is a case where we should focus on the court of appeal judgment and not the ICO's guidance and its interpretation of that judgment. New proposed snippet:

>I would like to ask you to reassess your response to my request in light of the fact that it has a serious purpose and value.
>A court of appeal judgment has suggested that a request for information which "would be of value to the requester, or to the public or any section of the public" should not generally be considered vexatious.  http://www.bailii.org/ew/cases/EWCA/Civ/2015/454.html

**Comment by garethrees** [comment]
@MyfanwyNixon could you take an initial look at this?

**Comment by MyfanwyNixon** [comment]
Yep, will add it to this sprint.

**Comment by MyfanwyNixon** [comment]
### tl;dr:

- The snippet Richard has identified actually comes when the user answers NO to 'Is the information you are requesting 'trivial'?' 
- However, this question comes before 'Has the authority stated that providing the information you have asked for would cause them a disproportionate or unjustified level of disruption, irritation or distress, or 'place an unreasonable burden' (which generates its own snippets) so people may be using it more often for that reason.
- We could think about changing the order of the questions accordingly (ie ask about the level of disruption before asking whether the matter is trivial; the ordering is not interdependent); this is something a developer would need to do as I can't do it through the [snippets tool](https://www.whatdotheyknow.com/admin/snippets?page=6).
- Details of all the questions and the snippets they lead to are below - I suggest Richard comments on these before we take any action on the re-ordering.
- I can do rewording of existing snippets but s I say, anything more detailed needs to be done by a developer.

As a side note, this query brought up the fact that it's not that easy to locate a specific snippet within the snippets tool!

### Longer answer:

This advice is displayed where a Section 14 exemption appears to have been applied, and as a response to the following decision tree (where the user has marked first No and then Yes):

- Is the information you are requesting 'trivial'? The ICO states that a vexatious request may be one where "the matter being pursued by the requester is relatively trivial and the authority would have to expend a disproportionate amount of resources in order to meet their request". It also may be "frivolous" ("inane or extremely trivial subject matter").

   - **NO**:
Can you demonstrate the 'serious purpose and value' (ICO) of this information to yourself or to the public, and that this justifies any disruption, irritation or distress on behalf of the authority in providing it?

      - **YES**:
A Court of Appeal case stated that a request may not be vexatious if "the information sought would be of value to the requester or to the public or any section of the public".
**ACTION >** Consider requesting an internal review, citing the ICO guidance and stating the value this information would have if released. 
     - **NO**:
It sounds like the exemption has been correctly applied: this is an improper use of FOI (and of WhatDoTheyKnow).  

The question that you identify, Richard, kicks off this decision tree:

- Has the authority stated that providing the information you have asked for would cause them a disproportionate or unjustified level of disruption, irritation or distress, or 'place an unreasonable burden' on them?
**Advice**: The ICO advises that "Public authorities must keep in mind that meeting their underlying commitment to transparency and openness may involve absorbing a certain level of disruption and annoyance" [italics ours].
   - **YES**:
 Have they provided any arguments or evidence supporting the assessment?
 
     - **NO**:
ACTION >  Reply and ask the authority to provide the arguments or evidence behind their assessment. _This leads to snippet 15 (see below)_
 
     - **YES**:
Has this assessment been based solely on cost? 
   
        - **YES**:
**ACTION** > The authority potentially should have cited Exemption 12, 'Appropriate limit'. Consider responding to clarify, and taking the steps advised for this exemption instead. _This leads to snippet 16 (see below)_ 
Has the authority suggested ways in which your request might be made less burdensome? This is not mandatory, but is 'best practice', according to the ICO.
    
       - **NO**
**ACTION** > Reply, citing ICO guidance and asking for such suggestions. _This leads to snippet 17 (see below)_ 

   - **NO**:
Has this assessment been based on the consequences of the release, rather than the act of responding? ICO guidance states that Section 14(1) is concerned with the nature of the request rather than the consequences of releasing the requested information. 

      - **YES**:
**ACTION** > Request an internal review, citing this guidance.  _This leads to snippet 18 (see below)_ 

      - **NO**: Consider objectively whether it is a reasonable assessment that responding to your request would require 'a disproportionate or unjustified level of disruption, irritation or distress, or place an unreasonable burden' on the authority. 
If you decide that it is not, gather your own arguments or evidence against this assessment, then include them when you ask for an internal review.

**Snippet 15:**
**The authority has refused your request on the grounds that it is vexatious, but has not provided reasons why responding to your request would cause them a disproportionate or unjustified level of disruption, irritation or distress, or 'place an unreasonable burden' on them. [clarification]**
I would be grateful if you could provide the reasoning behind your assessment that this was a vexatious request. My intention was not to cause disruption or annoyance, and seeing the reasoning applied would help me to understand how to reframe my request accordingly.
As you may be aware, the ICO advises that "public authorities must keep in mind that meeting their underlying commitment to transparency and openness may involve absorbing a certain level of disruption and annoyance". 
Additionally, the ICO advise that best practice for authorities applying this exemption is to suggest ways in which a request may be made less burdensome (see https://ico.org.uk/media/for-organisations/documents/1198/dealing-with-vexatious-requests.pdf).

**Snippet 16:**
**The authority have applied a section 14 exemption ('vexatious requests') but you think it should have more correctly been section 12 ('exceeds cost limit'). [clarification]**
I would be grateful if you could review whether this is the correct exemption to have been applied: would section 12, 'Cost of compliance exceeds appropriate limit' have been more appropriate? If so, please reassess my request accordingly.

**Snippet 17**
**The authority has refused your request on the grounds that it is vexatious, but has not suggested ways in which your request might be made less burdensome.  [clarification]****
The ICO advise that best practice for authorities applying this exemption is to suggest ways in which a request may be made less burdensome (see https://ico.org.uk/media/for-organisations/documents/1198/dealing-with-vexatious-requests.pdf). I would be grateful if you would make such suggestions so that I can reframe my request.
May I also take this opportunity to assure you that my request was not intended to be vexatious. [_As per another ticket - which I can't currently find but feel free to edit this to add it in if you know which one I mean - this advice did originally also say 'Include your reasons for having made the request' - but it was feared that users would miss this advice and send it off as part of their request for a review, so we dropped all such instructions_).

**Snippet 18**
**The authority has refused your request on the grounds that it is vexatious, but this assessment has been based on the consequences of the release, rather than the act of responding. [internal review]**
ICO guidance states that Section 14(1) is applied because of the nature of the request rather than the consequences of releasing the requested information (see https://ico.org.uk/media/for-organisations/documents/1198/dealing-with-vexatious-requests.pdf).
_As above, this originally also contained advice to add more details._


For Richard's info:
You might find it useful to see [this document](https://docs.google.com/document/d/15X685RGqRE_290ocZSc3tKw6sXKEyzhx2IWS3F2fdDQ/edit) explaining how the snippets were created/input.

[This is the document](https://docs.google.com/document/d/1IJnWbzoCGFiHYZUhG__AbP-6ReDvx8q3BCwvN_mglXk/edit) where all snippets were drafted.
